tensorflow-keras/src/main/java/org/tensorflow/keras/utils/TypeUtils.java

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.keras.utils;
+
+import org.tensorflow.DataType;
+import org.tensorflow.types.TBfloat16;
+import org.tensorflow.types.TBool;
+import org.tensorflow.types.TFloat16;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+import org.tensorflow.types.TInt32;
+import org.tensorflow.types.TInt64;
+import org.tensorflow.types.TString;
+import org.tensorflow.types.TUint8;
+
+
+/**
+ *
+ * @author Jim Clarke
+ */
+public class TypeUtils {
```

Comment:
Should we move all of this onto `DataType`? It feels like these would be better as instance methods.

Reply:
I agree

Reply:
In the different data types, the String names are raw strings, e.g. 
` DataType<TInt32> DTYPE = DataType.create("INT32", 3, 4, TInt32Impl::mapTensor);`
should we create a public static field?

`public static final String NAME = "INT32";`

Then the name can be accessed as `TInt32.NAME` rather than repeating the raw strings in the  DataType source.

Reply:
I think that's a good idea.

Reply:
I have moved this functionality to DataType and its instances (e.g TFloat32). This change will be added to this PR.